### PR TITLE
Allow to change the API domain via Config

### PR DESCRIPTION
**Pull request by Rafficer**
Just a simple addition that allows a user to set the preferred API in the config file

### CODE REVIEW

File: protonvpn_cli/utils.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 def call_api(endpoint, json_format=True, handle_errors=True):
     """Call to the ProtonVPN API."""
 
-    api_domain = "https://api.protonvpn.ch"
+    api_domain = get_config_value("USER", "api_domain").rstrip("/")
```
Comment (kaplun) [inline protonvpn_cli/utils.py:27]: What other sections are there? is `USER` the most appropriate?

File: protonvpn_cli/cli.py [hunk 2]
```diff
@@ -142,6 +142,7 @@ def cli():
         check_init()
         configure_cli()
     elif args.get("refresh"):
+        check_init()
```
Comment (kaplun) [inline protonvpn_cli/cli.py:145]: Why this `check_init()` call? Is it related to the commit?

File: protonvpn_cli/utils.py (hunk 1 above)
Comment (Rafficer) [reply protonvpn_cli/utils.py:27]: "metadata"

It should've been "Settings" and "metadata", but it's too late now :P

File: protonvpn_cli/cli.py (hunk 2 above)
Comment (Rafficer) [reply protonvpn_cli/cli.py:145]: Not necessarily, I just noticed that this is missing while testing.

Using refresh without an initialized profile throws an error.

**Pull request merged**